Example/Podfile

```
@@ -4,8 +4,8 @@ platform :ios, '10.0'
 
 target 'PrimerSDK_Example' do
   pod 'PrimerSDK', :path => '../'
-  pod 'Primer3DS'
-  pod 'PrimerKlarnaSDK'
+#  pod 'Primer3DS'
+#  pod 'PrimerKlarnaSDK'
```

We need to keep these as they are needed to perform BrowserStack tests.
We could think of starting to ignore files - I'll tell you something I've got in mind later today.
```suggestion
pod 'Primer3DS'
pod 'PrimerKlarnaSDK'
```